Spigot-Server-Patches/0333-Add-entity-count-cache.patch

```diff
@@ -0,0 +1,172 @@
+From 4f16c3445f22654d43ca5b2006f45fac0e17203b Mon Sep 17 00:00:00 2001
+From: Colin Godsey <crgodsey@gmail.com>
+Date: Wed, 8 Aug 2018 10:10:06 -0600
+Subject: [PATCH] Add entity count cache
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityInsentient.java b/src/main/java/net/minecraft/server/EntityInsentient.java
+index 7b64ec27c..55606756e 100644
+--- a/src/main/java/net/minecraft/server/EntityInsentient.java
++++ b/src/main/java/net/minecraft/server/EntityInsentient.java
+@@ -42,6 +42,7 @@ public abstract class EntityInsentient extends EntityLiving {
+     public float[] dropChanceArmor;
+     // public boolean canPickUpLoot; // CraftBukkit - moved up to EntityLiving
+     public boolean persistent;
++    public boolean countsAgainstSpawnLimit = true; // Paper
```

Comment:
added this to take care of the persistent inconsistencies. gets set to false based on whether its persistent went first "counted"